### DISCUSSION TITLE
solid start production build contains references to source code

### DISCUSSION THREAD

**Discussion by huseeiin** [solidjs/solid-start#1634] [General]
i think not everyone would be comfortable with this. it can be somewhat shocking (and a little funny :rofl:) that your computer name appears in public js.

is this a bug? if so, i hope a fix is planned.

![image](https://github.com/user-attachments/assets/6ed2f586-d11a-4587-984b-d6b28e24d489)

(file name is **client-COIQl8vs.js**)

**Comment by juanrgm** [comment]
All available routes are also displayed.
